.circleci/config.yml

```diff
@@ -355,7 +355,6 @@ jobs:
               --var preview_mongo_admin_url=$PREVIEW_MONGO_ADMIN_URL --var preview_mongo_url=$PREVIEW_MONGO_URL --var preview_database_url=$PREVIEW_DATABASE_URL \
               --var preview_okta_domain=$PREVIEW_OKTA_DOMAIN --var preview_okta_api_key=$PREVIEW_OKTA_API_KEY --var environment=Preview \
               --var preview_tealium_tag=$PREVIEW_TEALIUM_TAG --var preview_tealium_env=$PREVIEW_TEALIUM_ENV \
-              --var preview_ld_api_key=$PREVIEW_LD_API_KEY --var preview_ld_client_id=$PREVIEW_LD_CLIENT_ID \
```

Comment:
why did you need to take these out? I know we won't need LD_CLIENT_ID on the server, but we do need LD_API_KEY

Reply:
Launch Darkly isn't installed/enabled until provisioning. Launch Darkly is not set up on the AMI, so having the variables is unnecessary. We could add them to the AMI, but it would be a redundant and it still wouldn't be used until provisioning.